plugins/scm.js

```diff
@@ -82,6 +83,10 @@ const PARSE_URL = Joi.object().keys({
     token
 }).required();
 
+const CAN_HANDLE_URL = Joi.object().keys({
+    scmUri
```

Comment:
`canHandleUrl` takes scmUri as an argument, and returns boolean value which represents if the module can handle the scmUri. It's for determining which module to pass from scm-router to.